freqtrade/edge/__init__.py

```diff
@@ -203,6 +203,22 @@ def adjust(self, pairs) -> list:
 
         return self._final_pairs
 
+    def accepted_pairs(self) -> list:
+        """
+        return a list of accepted pairs along with their winrate, expectancy and stoploss
+        """
+        final = []
+        for pair, info in self._cached_pairs.items():
```

Comment:
Side note: a list comprehension would perform better, but with this conditional it would be too messy.